package/Dockerfile

```diff
@@ -86,6 +86,11 @@ RUN zypper --non-interactive install --no-recommends \
   && find /usr/lib/locale/* -maxdepth 1 | grep -v -E "(en_US|cs_CZ|es_ES|de_DE|C.utf8)" | xargs rm -rf \
   && find /usr/share/locale -name "*.mo" -delete
 
+# fail when there are multiple Ruby interpreters installed
```

Comment:
ah, the comment explains *What*, but let's add *Why*: 
```suggestion
# fail when there are multiple Ruby interpreters installed
# Because it means one part of YaST wanted a different version than the other
# and requires/imports will fail on the version mismatch.
```